src/ox_modules/module-mob.js

```diff
@@ -276,6 +276,11 @@ export default class MobileModule extends WebDriverModule {
      * @summary Ends the current session.
      */
     async dispose(status) {
+
+        if (!status) {
+            throw new Error('Status field is required and should be one of passed, failed or canceled');
```

Comment:
Should throw OxygenError with SCRIPT_ERROR instead of throwing generic Error, similiarly how we do for all other situations with invalid arguments.

Also documentation for this method is not updated.

And we should probably accept only `passed` and `failed`. `cancelled` is in internal status which is triggered only when stopping currently running and thus shouldn't be settable by user?

Reply:
Fixed